web-app/src/services/holidayService.js

```diff
@@ -24,10 +24,13 @@ export const approveHoliday = eventId => {
   return axios.put('/holidays/approveHoliday', { eventId });
 };
 
-export const rejectHoliday = eventId => {
+export const rejectHoliday = (eventId, message) => {
+  if (message) {
+    return axios.put('/holidays/rejectHoliday', { eventId, message });
+  }
   return axios.put('/holidays/rejectHoliday', { eventId });
```

Comment:
Not sure I like a check on the message to be honest, my personal preference should be if no message is sent there it's default set to empty. The backend should do the crux of the work after that, we just send them the details. Happy to discuss if anyone doesn't agree. 

`export const rejectHoliday = (eventId, message='') => {
  return axios.put('/holidays/rejectHoliday', { eventId, message });
};`

Reply:
Or we could handle this upstream a bit more, in the actual component when passing the data through.

Reply:
@eoinmcafee I'd like your thoughts on this also above.